Macco-Academy/chirp
# 185345741/create user account

### What does this PR do?
This PR allows creation of new User Account

### Description of tasks to be completed
* save user profile image to FirebaseStorage
* store user information in users table in firestore
* save user object locally in UserDefaults
* redirect to HomeTabViewController

### How to manually test
* enter phoneNumber on Login Page
* enter 6 digit code on OTP Page
* enter name and select profile image on Register Page
* hit 'Create Account' button
* navigate to Firestore to ensure user creation

### Any background context you want to provide
You can use test phone number '+18007779999' and test code '000000'. However ensure first it's not already in DB by visiting Firestore

### Screenshots
![Create Account Demo](https://github.com/Macco-Academy/chirp/assets/32524106/abdca42f-6974-4197-af65-43198ac57eae)
